src/IIIF/IIIF/Auth/V2/AuthAccessToken2.cs

```diff
@@ -9,10 +9,10 @@ public class AuthAccessToken2 : JsonLdBase
     public string Type => nameof(AuthAccessToken2);
 
     [JsonProperty(Order = 10, PropertyName = "messageId")]
-    public string? MessageId { get; set; }
+    public string MessageId { get; set; }
```

Comment:
maybe use `= null!`?

Reply:
Yeah - I've created #38 as there are a tonne of compiler warnings that need handled.